packages/frontend/src/components/InboxItem/InboxItem.tsx

```diff
@@ -100,9 +100,12 @@ export const InboxItem = ({
   const { inSelectionMode } = useSelectedDialogs();
   const { t } = useTranslation();
   const disableBulkActions = useFeatureFlag<boolean>(FeatureFlagKeys.DisableBulkActions);
+  const location = useLocation();
 
   const hideSummaryAndMeta = !summary || viewType === 'archive' || viewType === 'bin';
 
+  const linkToWithParams = `${linkTo}${location.search}`;
```

Comment:
Could I do suggest a little improvement here?

InboxItem should probably be so dumb as possible, and rather, let this linkTo + location.searche be lifted to `Inbox`. What do you think of the following:

1. remove the prop `linkTo` from `InboxItemInput` (both fra the interface and from the item-mapping in `useDialogs.tsx`.
2. In `Inbox.tsx` where the items are mapped into  `InboxItem`, build the `linkTo` prop with following pattern `/inbox/${item.id}/${location.search}`  
3. Do the same for `InboxItem` used in `SearchDropdown`